cdc/sink/mysql_params.go

```diff
@@ -35,6 +35,8 @@ const (
 	// expose these two variables for redo log applier
 	DefaultWorkerCount = 16
 	DefaultMaxTxnRow   = 256
+	maxWorkerCount     = 1024
```

Comment:
how about prefix with `default` to uniform the naming convention?

can you add some comment about why chose these values.

Reply:
This is not the default value.